src/convolution-output.c

```diff
@@ -458,14 +455,32 @@ enum nnp_status nnp_convolution_output(
 		case nnp_convolution_algorithm_ft8x8:
 			input_transform_function = nnp_hwinfo.transforms.fft8x8_and_stream;
 			kernel_transform_function = nnp_hwinfo.transforms.fft8x8_and_stream;
-			output_transform_function = nnp_hwinfo.transforms.ifft8x8_with_bias;
+			switch (activation) {
+				case nnp_activation_relu:
+					output_transform_function = nnp_hwinfo.transforms.ifft8x8_with_bias_with_relu;
+					break;
+				case nnp_activation_identity:
+					output_transform_function = nnp_hwinfo.transforms.ifft8x8_with_bias;
+					break;
+				default:
+					goto cleanup;
```

Comment:
`nnp_convolution_output` should return an error code if `activation` has unknown value. I suggest to check activation inside `validate_convolution_arguments`. Then in these switch statements you write `NNP_UNREACHABLE;` to indicate that this case never happens. Compiler will use it for optimization.